dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/validation/TrackerImportValidationContext.java

```diff
@@ -201,16 +200,6 @@ public ProgramStageInstance getProgramStageInstance( String event )
         return bundle.getPreheat().getRelationship( bundle.getIdentifier(), relationship );
     }
 
-    public CategoryOptionCombo getCategoryOptionCombo( String id )
```

Comment:
The idea is to remove this class at some point, right?

I am not very familiar with the optionCombo/categoryCombo code but I thought one of the goal of this fix was to move the caches to the Preheat.
Maybe after this PR you have a much better understanding if this is possible and in case how to do it

Reply:
Yes :) I talked about it with Stian and we agreed to to this in a follow up https://jira.dhis2.org/browse/DHIS2-12623
This fix was already quite complex. I think we have a good set of tests now and at least another dev with more understanding of this 😋